add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -594,6 +595,16 @@ var LoopRoomsInternal = {
     return gGetAllPromise;
   },
 
+  getNumParticipants: function(roomToken) {
```

Comment:
We also need xpcshell unit tests for the changes in this file; specifically one that fails without these changes and passes with it.